azurelinuxagent/common/protocol/wire.py

```diff
@@ -692,6 +715,16 @@ def update_certs(self, goal_state):
         self.save_cache(local_file, xml_text)
         self.certs = Certificates(self, xml_text)
 
+    def update_remote_access_conf(self, goal_state):
+        if goal_state.remote_access_uri is not None:            
+            xml_text = self.fetch_config(goal_state.remote_access_uri, self.get_header_for_cert())
+        else:
+            logger.verbose("{0}: remote access uri is empty.".format(sys._getframe().f_code.co_name))
```

Comment:
Why is your verbose message reaching into sys._getframe()?  Is this necessary, or was it only for debugging?

Is it worth it to have this if/else in the code, or is it sufficient to just have an if.